### ISSUE TITLE
Peaks Site - Final To Do List

### ISSUE THREAD

**Issue by mountaingoat69** [WebCab-Design/peaksrecovery.com#3]
@TheBurnsby as we spoke about, let's consider moving Verify Insurance section to just a single tab like Symposium "Registration" button.... I think this looks pretty good, can we add a line going underneath slider section to that verify insurance button? Similar to what it looks like on Symposium site? Please and thank you. 

Let's also tighten up spacing between slider image and three verbiage areas below slider.

Also, not sure what your thoughts are on this, but the header area from menu bar up looks a bit chunky. Anyway to better separate those sections or not make it looks so blockish for that area? Probably makes no sense to either of you haha, just let me know if it doesn't and I'll expand on this. 

Finish staff page. Please just add myself and Corinne to that page and make sure bio features are working. 

Add new Contact Us form to pages: Contact, all mental health and addiction treatment pages, Mens program, womens program, Treatment Philosophy pages. 

Add model of care section to home page that I showed you @TheBurnsby 

Add Gender Specific video from youtube to Gender Specific Page. Let's add "A Day in the LIfe" video to Life skills page; and add Why we are Different Video maybe next to Model of Care image @TheBurnsby ? Not sure how that would look, but worth a shot. Otherwise, the only other place I can think to put it is on the admissions page. Let me know what you guys think. 

I have all logos for TPRC, SPRC, PRS, and PRC inside cloudinary. Let's populate those in the footer section. I'm not attached to the footer color so if we need to change that to accommodate the logos, let's do it! 

Speaking of logos, the Mens Program and Women's Program pages are technically going to be landing pages for web campaigns. That being said I'd like to have TPRC's logo on the mens page and SPRC's logo on the womens page. Not sure what it would look like to add these to menu bar for those specific pages, but if too complicated or not worth doing, maybe find a way to place on page somewhere else. Maybe we need to create unique links for these pages too like triplepeaks.peaksrecovery.com? Just a thought. 

I've also been uploading Peaks Facility photos to cloudinary in their unique folders. I have the two mens homes in their presently and will have the remaining photos in cloudinary by tomorrow morning (1/6/16) for office and women's housing.

**Comment by TheBurnsby** [comment]
@AlexanderElias 
Can you tackle these items? The home image should be a slider maybe first half could be welcome text and second hald could be one of the videos?

Add Joint Commission logo to slider section. Also, this section is my least favorite section haha. I know why we did it, but I don't like the background color running all the way across the slider.

Pages that need their own, unique header image is "Men's Program", "Women's Program", and "Staff".

Admissions pages, Treatment Philosophy pages, Addiction Treatment pages, and Mental Health will all have their own unique header image section, however, all tabs under Treatment Philosophy, for example, will have the image. If that makes sense?

**Comment by AlexanderElias** [comment]
@mountaingoat69 @TheBurnsby 

Pushed those updates for the features in jons comment. It looked like some of the others brandon mentioned was fixed. Let me know if there is any thing else missing.

For the top images on each page I have added the following code. The images still need to be picked and urls need to be added. But all the pages with top images mentioned should have that code. 
```HTML
<style>
	.main-image {
		background-image: url("/path/to/image/required");
	}
</style>

<div class="main-image"></div>
```

Slider is implemented on the home page it also needs an image or text content.
